python/pyarrow/tests/test_fs.py

```diff
@@ -621,12 +643,13 @@ def test_filesystem_from_path_object(path):
 
 
 @pytest.mark.s3
-def test_filesystem_from_uri_s3(minio_server):
+def test_filesystem_from_uri_s3(s3_connection):
```

Comment:
is there a reason this doesn't need the s3_server fixture?

Reply:
It needs it, running that case alone fails.